### PR TITLE
Move config options to file instead of hardcoding.

### LINKED ISSUE #44: Set up decrediton config files
Currently there are some hardcoded paths/values to connect to grpc client.

These should be handled via config files instead.

### PR DESCRIPTION
**Pull request by jcvernaleo**
Closes #44

### CODE REVIEW

**Comment by jcvernaleo** [comment]
This is still WIP.  Do not review yet.

**Comment by jcvernaleo** [comment]
This is no longer WIP.  Ready for review/merge.

File: app/index.js [hunk 1]
```diff
@@ -7,11 +7,21 @@ import { syncHistoryWithStore } from 'react-router-redux';
...
 import './bootstrap.min.css';
+import { getCfg } from './config.js';
+
+var cfg = getCfg();
+
+var grpcport = "";
+if (cfg.network == "testnet") {
+  grpcport = cfg.wallet_port_testnet;
+} else {
+  grpcport = cfg.wallet_port_testnet;
```
Comment (dajohi) [inline app/index.js]: should be ```cfg.wallet_port;``` ??

File: app/actions/WalletLoaderActions.js [hunk 2]
```diff
@@ -228,19 +229,17 @@ function startRpcSuccess() {
...
-    certPath = path.join(process.env.LOCALAPPDATA, 'Dcrd', 'rpc.cert');
-  } else if (os.platform == 'darwin') {
-    certPath = path.join(process.env.HOME, 'Library', 'Application Support',
-    'Dcrd', 'rpc.cert');
+  var cfg = getCfg();
+  var rpcport = "";
+  if (cfg.network == "testnet") {
+    rpcport = cfg.daemon_port_testnet;
+  } else {
+    rpcport = cfg.daemon_port_testnet;
```
Comment (dajohi) [inline app/actions/WalletLoaderActions.js]: ```cfg.daemon_port;``` ?

Comment (jcvernaleo) [inline app/actions/WalletLoaderActions.js]: oops.  fixed

File: app/index.js (hunk 1 above)
Comment (jcvernaleo) [inline app/index.js]: and fixed this one.

File: app/main.development.js [hunk 3]
```diff
@@ -24,6 +25,11 @@ process.argv.some(function(element) {
   }
 });
```
Comment (jcvernaleo) [inline app/main.development.js]: Removed.

File: app/main.development.js [hunk 4]
```diff
@@ -17,12 +18,10 @@ if (process.env.NODE_ENV === 'development') {
...
 }
 
-process.argv.some(function(element) {
-  if (element === '--testnet') {
-    mainNet = false;
-    return true;
-  }
-});
+var cfg = getCfg();
+if (cfg.network == "testnet") {
```
Comment (dajohi) [inline app/main.development.js]: delete this if block and the mainNet variable.

Comment (jcvernaleo) [inline app/main.development.js]: done

File: app/main.development.js [hunk 5]
```diff
@@ -41,9 +40,9 @@ function appDataDirectory() {
 
 function RPCWalletPort() {
   if (mainNet == true) {
```
Comment (jcvernaleo) [inline app/main.development.js]: done

**Comment by alexlyp** [comment]
tACK on npm run dev

**Pull request merged**